process/smartContract/builtInFunctions/claimDeveloperRewards.go

```diff
@@ -22,6 +22,11 @@ func NewClaimDeveloperRewardsFunc(gasCost uint64) *claimDeveloperRewards {
 	return &claimDeveloperRewards{gasCost: gasCost}
 }
 
+// SetNewGasConfig is called whenever gas cost is changed
+func (c *claimDeveloperRewards) SetNewGasConfig(gasCost *process.GasCost) {
+	c.gasCost = gasCost.BuiltInCost.ClaimDeveloperRewards
```

Comment:
We need to protect `e.gasCost` with a mutex

Reply:
no need. there is no possibility to call the from different go routines

Reply:
it is protected all the way right now.